src/client/pfs.go

```diff
@@ -543,6 +566,7 @@ func (w *putFileWriteCloser) Write(p []byte) (int, error) {
 
 func (w *putFileWriteCloser) Close() error {
 	_, err := w.putFileClient.CloseAndRecv()
+	err = sanitizeErr(err)
 	return err
 }
 
```

Comment:
Could you just inline the `sanitizeErr` calls? Storing it in a variable each time seems like a pretty big waste of vertical space.